src/NodeApi/DotNetHost/NativeHost.cs

```diff
@@ -158,6 +163,8 @@ private JSValue InitializeFrameworkHost(Version minVersion, string managedHostPa
 
             // Create an "exports" object for the managed host module initialization.
             JSValue exportsValue = JSValue.CreateObject();
+            exportsValue.SetProperty("require", require);
```

Comment:
>exportsValue.SetProperty("require", require); [](http://example.com/codeflow?start=12&length=45)

Should we rather define a managed entry point and pass it as a normal parameter. It feels wrong that we add a new property to `exports`.

Reply:
It gets overwritten by the `require()` method defined by the `ManagedHost`. So this property is never visible to applications.